Week-1/day2/favMusician/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Arijit Singh Fan Webpage</title>
+</head>
+<body>
+  <header>
+    <h1>Arijit Singh Fan Webpage</h1>
+  </header>
+  <main>
+    <p>Arijit Singh (born 25 April 1987) is an Indian playback singer and music composer.[6][4][2] The recipient of several accolades including a National Film Award and seven Filmfare Awards, he has recorded songs in several Indian languages, predominantly Hindi and Bengali.</p>
```

Comment:
The <p> tag directly inside the <body> could be enclosed within a <section> or <article> for better semantic structuring.